### DISCUSSION TITLE
Use stackalloc for multiple instances of arrays of constant size

### DISCUSSION THREAD

**Discussion by Mrnikbobjeff** [dotnet/maui#22616] [Ideas]
### Description

Referencing  #11846
searching for 'new char[' / 'new int[' reveals several instances where we allocate tiny temporary char arrays on the heap instead of stackallocing them.
This [search](https://github.com/search?q=repo%3Adotnet%2Fmaui+new+char%5B&type=code) reveals multiple simple optimizable instances.

### Public API Changes

None

### Intended Use-Case

Removing unnecessary allocations

**Comment by MartyIX** [comment]
Why don't you simply create a PR for it?
